store/index.js

```diff
@@ -158,6 +158,19 @@ export const getters = {
         return mappedColumns;
     },
 
+    getMissingValues: (p_state) => (p_column) => {
```

Comment:
I think this is the inverse situation to https://github.com/neurobagel/annotation_tool/pull/353: 

We want to ask for "all missing values in any column assigned to a category", so the getter should take a category and then figure out what columns to get the missing values from. 

If you take a look at `annot-missing-values`, the returned value is expected to be an array of tuples (?) with the first entry being the columnName, and the second the missing Value. 

Not saying you need to stick to this exact format, but the general functionality should be as described.